ql/src/java/org/apache/hadoop/hive/ql/ddl/process/show/compactions/ShowCompactionsOperation.java

```diff
@@ -81,6 +81,8 @@ private void writeHeader(DataOutputStream os) throws IOException {
     os.write(Utilities.tabCode);
     os.writeBytes("Worker");
     os.write(Utilities.tabCode);
+    os.writeBytes("Queue Time");
```

Comment:
"Queue time" sounds like "time spent in queue." Something like "Entered queue" sounds more like a timestamp to me.

Reply:
How about enqueue time? Short, easy to understand...

Reply:
Sounds good to me!